tests/lib/prepare.sh

```diff
@@ -375,8 +375,8 @@ EOF
 #!/bin/sh
 set -e
 # ensure we don't enable ssh in install mode or spread will get confused
-if ! grep 'snapd_recovery_mode=run' /proc/cmdline; then
-    echo "not in run mode - script not running"
+if ! grep -E 'snapd_recovery_mode=(run|recover)' /proc/cmdline; then
```

Comment:
You could use MATCH in this case

Reply:
It's outside of spread shell, so MATCH is not available.

Reply:
I think it should use the MATCH command in $TESTSLIB/bin

Reply:
In fact there are many MATCH in that file

Reply:
We can't use MATCH here, it's an inline document that ends up being part of a script started by systemd. It starts in line 374 and ends in line 423.